remove-tmp-files.sh

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-find /var/www/html/var/tmp -mindepth 1 -mtime +7 -delete
+find /var/www/html/var/tmp -mindepth 1 -mtime 7 -delete
```

Comment:
hm, doesn't this return files that are modified between 7 and 8 days? https://unix.stackexchange.com/a/174743